### DISCUSSION TITLE
NIST standardization "NIST IR 8413" vs changes in the current implementation related to algorithm definitions, parameters

### DISCUSSION THREAD

**Discussion by yerracs86** [open-quantum-safe/liboqs#1257] [Q&A] [answered]
Since NIST standardization has begun "NIST IR 8413", will there be any changes expected in the current algorithm definitions, parameters and operations.

**Comment by dstebila** [comment, marked as answer]
I'm not aware yet of any indication from NIST as to whether there will be tweaks to the selected algorithms or not.
